src/pages/Employees/Employees.tsx

```diff
@@ -9,7 +9,7 @@ import {
 import EmployeeTable from "./components/EmployeeTable";
 import { EmployeeCardSkeleton } from "./components/EmployeeCard/EmployeeCard";
 import Button from "../../components/Button";
-import EmployeeModal from "./components/EmployeeModal";
+import EmployeeModal from "./components/EmployeeModal/EmployeeModal";
```

Comment:
Why did the import path change?

Reply:
I was receiving an error saying that `EmployeeModal` did not have a default export